scene/gui/tree.cpp

```diff
@@ -3354,14 +3355,16 @@ void Tree::gui_input(const Ref<InputEvent> &p_event) {
 						int len = 0;
 						for (int i = 0; i < columns.size(); i++) {
 							len += get_column_width(i);
-							if (pos.x < len) {
-								emit_signal(SNAME("column_title_pressed"), i);
+							if (pos.x < static_cast<real_t>(len)) {
```

Comment:
Why a static cast? And furthermore, if there is a need for a static cast, why is there not one down below in a similar code doing similar stuff with the same types of data? But I don't think there is a need to begin with.

Reply:
It was discussed here: https://github.com/godotengine/godot/pull/64369#discussion_r945174667

Reply:
Well then the code below should also be changed.

Reply:
Oh, you mean because of the massive code duplication? :)     I will go add it there... :(

Reply:
Uh-huh :)

Reply:
changed via force push